src/Phinx/Migration/Manager.php

```diff
@@ -768,6 +769,58 @@ public function setSeeds(array $seeds)
         return $this;
     }
 
+    /**
+     * Get seed dependencies instances from seed dependency array
+     *
+     * @param OrderedSeedInterface $seed Seed
+     *
+     * @return AbstractSeed[]
+     */
+    protected function getSeedDependenciesInstances(OrderedSeedInterface $seed)
+    {
+        if (empty($this->seeds)) {
+            return [];
+        }
+
+        $dependenciesInstances = [];
+        $dependencies = $seed->getDependencies();
+        foreach ($dependencies as $dependency) {
+            foreach ($this->seeds as $seed) {
+                if (get_class($seed) === $dependency) {
```

Comment:
Sorry if I am wrong, but if people make a typo in the etDependencies() method, it will be discarded silently. Couldn't it be implemented by using `usort()` and the callback method? I think you want to sort seeds by a custom logic. In such case, I would use `usort()`.

Reply:
I can add exception if there is no dependency written in the function. About sorting seeds in custom logic, I can not imagine a case where it would be used. Why you want to order seeds in custom logic? And why write additional code if there is simple and suitable solution with dependencies...

Reply:
Sorry for my English. I didn't say I wanted to order seeds in custom logic. I meant that your implementation looks like that it sorts seeds by using two methods - `orderSeedsByDependencies()` and `getSeedDependenciesInstances()`. But if you have a method to compare seeds, you can do `usort($seeds, [$this, 'compareSeeds'])`.

Also, people cannot use `::class` syntax to avoid typos. Try `NonExisitentClass::class`. It would also returns `NonExisitentClass`.

Reply:
It could be implemented in many ways :). Please write me explicitly what should I fix, to merge this feature (I need it in production urgently...).

Reply:
OK. Then please don't add additional interface. Other points are minor things. It could be changed later if someone interests.